src/EnergyPlus/HVACManager.cc

```diff
@@ -891,7 +891,7 @@ namespace HVACManager {
 							}
 						} // humidity ratio not converged
 
-						if ( AirLoopConvergence( AirSysNum ).HVACTempNotConverged ) {
+						if( any( AirLoopConvergence( AirSysNum ).HVACTempNotConverged ) ) {
```

Comment:
@DeadParrot Do you have any thoughts on this?  I think you wrote a specific FArray_bool ```any``` function a while back.  Is that still true?